### DISCUSSION TITLE
v4.3 - Call for beta testers

### DISCUSSION THREAD

**Discussion by edwardtfn** [Blackymas/NSPanel_HA_Blueprint#1875] [General]
Hello NSPanel Community,

We are thrilled to invite you to the beta testing of NSPanel HA Blueprint v4.3, a milestone release that not only transitions to the ESP-IDF framework for improved performance and flexibility but also introduces the Utilities Dashboard for enhanced utility tracking, significant localization improvements, and an array of other enhancements to enrich your smart home experience.

**Key Highlights of v4.3.0:**
- **Utilities Dashboard**: Monitor essential utility metrics directly from your NSPanel.
- **Expanded Localization**: Enhanced user interface with user-defined decimal separators and translations of values for a global audience.
- **Comprehensive API Documentation**: Detailed service documentation to facilitate customization and integration.
- **Enhanced Personalization and Visibility**: From hardware button bars across all pages to user-defined decimal separators and translated values, v4.3.0 brings you a personalized experience.

We're particularly excited about the feedback on the **Utilities Dashboard** and the **expanded entity selection for Home and Climate pages**, combined with state_translated for a richer, more customized user experience.

**Joining the Beta Test:**
Dive into the [`beta`](https://github.com/Blackymas/NSPanel_HA_Blueprint/tree/beta) branch for all necessary files and instructions. For those new or needing a refresher on beta or other versions, our [Guide to Using Different Versions of This Project](https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/beta/docs/different_version.md) is a great resource.

**We're Listening:**
Share your testing experience directly in our GitHub Discussions. Whether it's a feature that enhanced your NSPanel's functionality, a bug that needs squashing, or a suggestion for future updates, every bit of feedback is invaluable. Here are some key areas we're keen to hear about:
- How did the **Utilities Dashboard** enhance your monitoring experience?
- Did the **expanded entity selection** for Home and Climate pages improve your control and visibility?
- How have the **localization improvements** impacted your use of the NSPanel?

**Feedback Submission:**
While we're not using polls, we encourage detailed feedback in the discussion area. Here’s how you can help:
- **Report issues** on our [Issues page](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues).
- **Suggest enhancements** or submit fixes via Pull Requests to the [`dev`](https://github.com/Blackymas/NSPanel_HA_Blueprint/tree/dev) branch.

Thank you for being an active part of the NSPanel HA Blueprint community. Your insights and contributions are crucial for the continuous improvement of this project. Let's make NSPanel HA Blueprint v4.3.0 the best version yet!

Happy testing!

[NSPanel HA Blueprint Team](https://github.com/Blackymas/NSPanel_HA_Blueprint)

**Comment by shing6326** [comment]
Hello @edwardtfn , thanks for your great efforts to always give the NSPanel_HA_Blueprint users with tones of new features and enhancement. I have a few suggestions that would like to share which based on my own opinion and use case only.
1. [Button and Entity pages] able to disable the page selector (dot) when there is only one page from button/entity pages.
2. [Entity page] enable the option to switch the automatic font size adjustment for entity page values on and off. My entity page values are highly related each others and it may contains some CJK characters which hard to calculate the width of the string, it is good to have the switch so that my entities may look consistent with all values.
3. [Entity page] allow users to disable the entity icons from entity pages
4. [Fan page] consider to merge the on and off buttons into single power button which similar to the light page.

**Reply by edwardtfn** [reply]
- #1883
- #1884
- #1885
- #1886

**Reply by edwardtfn** [reply]
Thanks for your feedback!!

**Reply by plasterer41** [reply]
I’ve no issues with the last dev version. I assume it is same as beta version Edward?

thanks again for all your hard work maintaining and updating this software 👍🏼

**Comment by edwardtfn** [comment]
v4.3 was released!

**Reply by edwardtfn** [reply]
- #1891
